static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -88,12 +86,12 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
               {feedbackItem.project.slug}
             </Flex>
 
-            {feedbackItem.replay_id ? (
+            {/* {feedbackItem.replay_id ? (
```

Comment:
leave this out and make use of the replay-count

Reply:
Lets not even edit this now. It will cause git conflicts with https://github.com/getsentry/sentry/pull/58732

Reply:
```suggestion
            {feedbackItem.replay_id ? (
```